react/react_intro/README.md

```diff
@@ -290,28 +309,24 @@ Returning to your browser, you should see the following image:
 
 ![feedImgNoCSS](./images/feedImgNoCSS.png)
 
-We have multiple components rendered on the page all at once!  Now we can add our own styling by configuring separate CSS files.
-
-# 5. CSS and React
+We have multiple components rendered on the page all at once! Now, to add styling.
 
-Because each of our components is its own separate module, we can create CSS files for each individual component.  This will help make it easier to work with individual components without worrying about the rest of the app.  Let's put a border around our `ContactList` and change the font to cursive.  First, we'll need to add the `className` to our `div`:
+# CSS and React
 
-`ContactList.js`
+Just like we can create separate JS files for each component, we can create separate CSS files for them, too. This will make it easier to style individual components without worrying about the rest of the app. Let's put a border around our `ContactList` and change the font to cursive. First, we'll need to add a class to our containing `div`:
 
-```js
-class ContactList extends React.Component {
-    render() {
-        return (
-            <div className='ContactList'>
-						...
-				)
-   }
+```jsx
+const ContactList = () => {
+  return (
+    <div className='ContactList'>
```

Comment:
@coreyladovsky do we want pascal casing for class names?

Reply:
I think this is good for React. Seems like the convention for react is PascalCase and Block__Element--Modifier

Reply:
We actually use camelCase in our application, but it's possible that's because we don't pass strings into our `className`s. I'm not too opinionated about this, but from what I've seen, camelCase is more common.